### ISSUE TITLE
v3: setoutput is deprecated

### ISSUE THREAD

**Issue by neilmunday** [py-cov-action/python-coverage-comment-action#118]
When using `py-cov-action/python-coverage-comment-action@v3` in a workflow, the follow warning is reported by Github:

```
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files.
```

In the v2 branch, commit https://github.com/py-cov-action/python-coverage-comment-action/commit/9c01da2f0ddbd0d6350b3d1ec93d7df0e121a21b was added to correct the issue, this appears to be missing in the v3 branch.

**Comment by kieferro** [comment]
Thank you for reporting the problem. This is strange. Actually, this problem was also fixed for `v3` in #84 and as far as I can see this problem does not exist in other runs. Is the run showing this message a bit older or is it possibly related to a specific commit in the v3 branch where the fix does not exist yet?

**Comment by neilmunday** [comment]
I think I've found and resolved the issue. I was using the `actions/download-artifact@v2` and `actions/upload-artifact@v2` actions in my `coverage` job which produced the error above (as well as NodeJS 12 deprecation errors).

I've since updated my workflow to use version 3 of both these actions and I no longer get any errors.
